src/main/resources/templates/pages/index.html

```diff
@@ -30,6 +30,7 @@
 	    </a>
 	    <ul class="dropdown-menu">
 			<li><a href="#/list/BY_NAME_AND_IV">By Name and IV</a></li>
+            <li><a href="#/list/BY_NUMBER_AND_IV">By # and IV</a></li>
```

Comment:
Seems like a problem with a mixed usage of spaces and tabs on the file which is making the editors (both IntelliJ and Visual Studio Code) recognize it as 4 spaces instead of tabs.